### DISCUSSION TITLE
Publication Dates Showing on Pages

### DISCUSSION THREAD

**Discussion by toddgrotenhuis** [endonend/Hains-Point#11] [Ideas]
It seems that dates should only show on blog posts, not Pages. Many page dates are either irrelevant or false (like the search page).

**Comment by endonend** [comment]
Fair point. I will consider making it optional in settings or potentially using the last updated date vs the publish date, if that's possible.

**Comment by endonend** [comment]
Next update will change pages to "last modified". I will consider making that optional in the future.

Starting to get concerned I'm adding too many settings to the plugin though! :-)

**Comment by endonend** [comment]
Going to convert this to a discussion so others can weigh in...
